spec/lib/url_cache_spec.rb

```diff
@@ -24,44 +24,93 @@
     end
   end
 
+  shared_examples "memcached exceptions" do |dalli_method_name|
+    describe "memcached exception handling" do
+      let(:dalli_client) { instance_double(Dalli::Client) }
+      let(:url_cache)    { described_class.new(cache: dalli_client) }
+
+      let(:error_message) { "An error has occured!" }
+      let(:dalli_error)   { dalli_error_class.new(error_message) }
+
+      before do
+        allow(dalli_client)
+          .to receive(dalli_method_name)
+          .and_raise(dalli_error)
+      end
+
+      context "when a Dalli::RingError is raised" do
+        let(:dalli_error_class) { Dalli::RingError }
+
+        it "retries a few times" do
```

Comment:
not sure you saw my comment in https://github.com/twingly/twingly-url_cache/pull/9#discussion_r261588141 concerning this, so I'll repeat it.

I think we should change "few" to something more "spec-y". It could state "it retries three times" instead. Or even make a variable that we make use of in text and in assertion.

Reply:
I must have missed that part, sorry.

> Or even make a variable that we make use of in text and in assertion.

That sounds good. The point of writing "a few" was just because I didn't want to hardcode a "3" in the text that could get out of date easily.

Reply:
> The point of writing "a few" was just because I didn't want to hardcode a "3" in the text that could get out of date easily.

Although I don't think we will change this very often :)

Reply:
f259f7a